sdk/cosmos/azure-cosmos/azure/cosmos/cosmos_client.py

```diff
@@ -241,29 +243,26 @@ def _get_database_link(database_or_id):
     def create_database(  # pylint: disable=redefined-builtin
         self,
         id,  # type: str
-        populate_query_metrics=None,  # type: Optional[bool]
+        populate_query_metrics=None,  # type: Optional[bool] # pylint:disable=docstring-missing-param
         offer_throughput=None,  # type: Optional[Union[int, ThroughputProperties]]
         **kwargs  # type: Any
     ):
         # type: (...) -> DatabaseProxy
         """
         Create a new database with the given ID (name).
-
-        :param id: ID (name) of the database to create.
+        :param str id: ID (name) of the database to create.
         :param offer_throughput: The provisioned throughput for this offer.
-        :paramtype offer_throughput: int or ~azure.cosmos.ThroughputProperties.
+        :type offer_throughput: Union[int, ~azure.cosmos.ThroughputProperties]
         :keyword str session_token: Token for use with Session consistency.
-        :keyword dict[str,str] initial_headers: Initial headers to be sent as part of the request.
+        :keyword Dict[str, str] initial_headers: Initial headers to be sent as part of the request.
         :keyword str etag: An ETag value, or the wildcard character (*). Used to check if the resource
             has changed, and act according to the condition specified by the `match_condition` parameter.
         :keyword ~azure.core.MatchConditions match_condition: The match condition to use upon the etag.
         :keyword Callable response_hook: A callable invoked with the response metadata.
         :returns: A DatabaseProxy instance representing the new database.
         :rtype: ~azure.cosmos.DatabaseProxy
         :raises ~azure.cosmos.exceptions.CosmosResourceExistsError: Database with the given ID already exists.
-
         .. admonition:: Example:
-
```

Comment:
I think these whitespaces might be necessary? You can download the rendered docs from the CI outputs in devops - might be worth just checking that the code snippets are rendering okay.

Reply:
yep I think you're right, seeing some weird stuff in a bunch of other places, will fix them now